Task topics for Opaque Integer Resoures

steveperry-53 opened the pull request:
@vishh, this is ready for tech review.

The purpose of this PR is to create dedicated Task topics for opaque integer resources.
For app developers: [Assign Opaque Integer Resources to a Container](https://deploy-preview-4363--k8sio.netlify.com/docs/tasks/configure-pod-container/opaque-integer-resource/).
For cluster administrators: [Advertise Opaque Integer Resources for a Node](https://deploy-preview-4363--k8sio.netlify.com/docs/tasks/administer-cluster/opaque-integer-resource-node/).

This PR is a companion to [PR 4158](https://github.com/kubernetes/kubernetes.github.io/pull/4158), and should not be merged until after PR 4158 is merged.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.kubernetes.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4363)
<!-- Reviewable:end -->

steveperry-53 commented:
@vishh, Instead of sending an HTTP PATCH request, could we use kubectl patch? After we answer that question, we should show a cleanup step that removes the dongle advertisement from the node.

ConnorDoyle commented:
@steveperry-53 I don't believe `kubectl patch` will work, although that would be really nice to get working. The reason is that `kubectl patch` uses the strategic merge patch strategy instead of plain old `application/json-patch+json`.

vishh commented on docs/tasks/administer-cluster/opaque-integer-resource-node.md:
````diff
@@ -0,0 +1,147 @@
+```yaml
+Capacity:
+ ...
+ pod.alpha.kubernetes.io/opaque-int-special-storage:  8
````
Or you could also advertize as `800GiB` and let users request arbitrary amounts. May be just mentioned that for completeness?

steveperry-53 replied:
Done.

vishh commented on docs/tasks/administer-cluster/opaque-integer-resource-node.md:
```diff
@@ -0,0 +1,147 @@
+
+{% capture overview %}
+
+This page shows how to specify opaque integer resources for a Node.
```
I want to highlight that this feature is `alpha` and that the API may change or be removed alltogether.

steveperry-53 replied:
Done.

steveperry-53 commented:
@vishh, Back to you. I addressed your comments in https://github.com/kubernetes/kubernetes.github.io/pull/4363/commits/5a3953096af2d584470ed632f35f992029c85b1b.
